analyses/methylation-summary/04-beta-tpm-correlation.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+
+
+"""
+04-beta-tpm-correlation.py
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Calculate representative probe-level correlations between RNA-Seq (TPM) and methylation (Beta) for patients who have samples in both datasets
+"""
+
+
+__author__ = ('Eric Wafula (wafulae@chop.edu)')
+__version__ = '1.0'
+__date__ = '15 May 2022'
+
+
+import re
+import os
+import sys
+import numpy as np
+import pandas as pd
+import pyreadr as rds
+
+
+# establish base dir
+root_dir = os.path.abspath("../../")
```

Comment:
```suggestion
root_dir = os.path.abspath("../../")
```

Can this be more explicit such as the finding git path in R?

Reply:
updated using the `gitpython` module

```
root_dir = git.Repo('.', search_parent_directories=True).working_tree_dir
```